source/change-streams/tests/README.rst

```diff
@@ -77,10 +77,16 @@ Pseudocode implementation of ``actual`` MATCHES ``expected``:
         Assert that actual[idx] MATCHES value
     Else if expected is a JSON object:
       For every key/value in expected
-        Assert that actual MATCHES value
+        Assert that actual[key] MATCHES value
     Else:
       Assert that expected equals actual
 
+The expected values for ``result.success`` and ``expectations`` are written in Extended JSON. Drivers may adopt any of the following approaches to comparisons, as long as they are consistent:
```

Comment:
I have no issues with pedantry.  It's a spec!

We don't need to note it because an extended JSON parser must be able to parse either without being told which it is:

> By default, a parser MUST accept values in either Canonical Extended JSON format or Relaxed Extended JSON format as described in this specification. A parser MAY allow users to restrict parsing to only Canonical Extended JSON format or only Relaxed Extended JSON format.